LittleToySourceGenerator.Tests/GenerateEventDataTests.cs

```diff
@@ -35,24 +35,25 @@ public partial struct Position3Data : Unity.Entities.IComponentData
 using Unity.Mathematics;
 using Plugins.basegame.Events;
 using DOTSNET;
+using Unity.Collections;
 
 public partial struct Position3Data
 {
     public bool IsDirty { get; set; }
-    public Position3Data Update(Unity.Mathematics.float3 value)
+    public Position3Data Update(float3 value)
     {
         if(Value.Equals(value)) return this;
-
+        
```

Comment:
This is changes in test to account for how CsCodeGenerator produce code. Hard to keep it intact.